CHANGELOG.MD

```diff
@@ -1,6 +1,10 @@
 # Change Log
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/).
```

Comment:
We're not using semantic versioning anymore. I personally also find "All notable changes to this project will be documented in this file." to be redundant